src/common/settings.js

```diff
@@ -1,39 +1,38 @@
 'use strict';
 
 const fs = require('fs');
+const deepmerge = require('deepmerge').default;
+
 const settingsVersion = 1;
+const baseConfig = require('./config/base.json');
+const overrideConfig = require('./config/override.json');
 
 function merge(base, target) {
   return Object.assign({}, base, target);
 }
 
+function deepMergeArray(dest) {
+  return dest;
+}
+
 function loadDefault(version, spellCheckerLocale) {
```

Comment:
Please test how `loadDefault()` works. Possibly `require('override.json')` doesn't match for writing tests because its static file.

Reply:
@yuya-oc do you want an automated test?

We followed the exact same style as the mobile app in terms of config and override. Possibly would like to match that to start with and then diverge?

Reply:
If possible. At least, I want that existing tests should be passed.
Unfortunately I'm not sure about mobile app source code. Would you tell me the part?